quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -170,11 +192,13 @@ public String apply(String arg) {
             }
         });
 
-        cmd.execute(configArgsList.toArray(new String[0]));
+        exitCode = cmd.execute(configArgsList.toArray(new String[0]));
 
         if(!isDevMode()) {
             cmd.getOut().printf("Next time you run the server, just run:%n%n\t%s %s %s%n%n", Environment.getCommand(), Start.NAME, String.join(" ", getSanitizedRuntimeCliOptions()));
```

Comment:
Does it make sense to only show this message when the exit code is success?

Reply:
done